src/main/java/ailurus/Parser.java

```diff
@@ -0,0 +1,95 @@
+package ailurus;
+
+import ailurus.task.*;
+
+public class Parser {
+
+    /**
+     * Parse message and return command
+     *
+     * @param message message to be parsed
+     * @return command that is passed
+     */
+    public static String parse(String message) {
+        return message.split(" ")[0];
+    }
+
+    /**
+     * Parse message and return string after command
+     *
+     * @param message message to be parsed
+     * @return message string that is after command
+     */
+    public static String parseMessage(String message) {
+        int index = message.indexOf(" ");
+        if (index > 0) {
+            return message.substring(index + 1);
+        } else {
+            return "";
+        }
+    }
+
+    /**
+     * Parse data stored in hard disk by line
+     *
+     * @param data line of data to be parsed
+     * @return task that is parsed from line
+     */
+    public static Task parseData(String data) {
+        String[] dataArr = data.split("[|]");
+        switch (dataArr[0]) {
+            case "T":
+                if (dataArr.length < 3) {
```

Comment:
perhaps use a variable "lens" to store the dataArr's length to avoid recalculating?